packages/nextjs/src/config/webpack.ts

```diff
@@ -86,18 +86,17 @@ export function constructWebpackConfigFunction(
         // if `pages` is present in the root directory"
         // - https://nextjs.org/docs/advanced-features/src-directory
         const shouldIncludeSrcDirectory = !fs.existsSync(path.resolve(projectDir, 'pages'));
+        const pagesDirectory = shouldIncludeSrcDirectory // We're intentionally not including slashes in the paths because we wanne let node do the path resolution for Windows
```

Comment:
Less than l: I would remove the comment, don’t think that context is needed.
```suggestion
        const pagesDirectory = shouldIncludeSrcDirectory
```